### DISCUSSION TITLE
[Modules/TensorMechanics/Master] in tests and examples

### DISCUSSION THREAD

**Discussion by jmeier** [idaholab/moose#28691] [Q&A Modules: Solid mechanics] [answered]
Dear Moose Community,

As far as I understand https://github.com/idaholab/moose/issues/25584, [Modules/TensorMechanics/...] should/has been renamed to [Modules/SolidMechanics/...]. In some of the examples and tests the old [Modules/TensorMechanics/...] is still showing up. I assume these examples/tests are still to be renamed?

If this is the case and you would like me to do so, can I try a PR that renames at least some of these examples/tests?

Disclaimer: I suspect that this will reduce the test coverage somewhat.

**Comment by GiudGiud** [comment, marked as answer]
Hello

It was renamed to Physics/SolidMechanics. The "Modules/" is being abandoned in favor of grouping the large actions under Physics.
A PR is welcome if you have time.

Coverage should stay the same, the old syntax has been registered to the renamed action. Under the hood, the same code is used in both cases.
